Software/Python/grove_mini_motor_driver/grove_mini_motor_driver.py

```diff
@@ -0,0 +1,249 @@
+from scipy.interpolate import interp1d
```

Comment:
@RobertLucian : Do we need scipy for interpolate.  Also, I am facing a problem when installing scipy on R4R with pip.

No scipy
```
 Traceback (most recent call last):
   File "driver_example.py", line 34, in <module>
     from grove_mini_motor_driver import MiniMotorDriver
   File "/home/pi/Desktop/test/GrovePi/Software/Python/grove_mini_motor_driver/grove_mini_motor_driver.py", line 1, in <module>
     from scipy.interpolate import interp1d
ImportError: No module named scipy.interpolate
```

Problem when installing scipy
```
Command /usr/bin/python -c "import setuptools, tokenize;__file__='/tmp/pip-build-woKenB/scipy/setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record /tmp/pip-nQjRmB-record/install-record.txt --single-version-externally-managed --compile failed with error code 1 in /tmp/pip-build-woKenB/scipy
Storing debug log for failure in /root/.pip/pip.log

```